python/dgl/cuda/nccl.py

```diff
@@ -210,4 +210,14 @@ def size(self):
         """
         return self._size
 
+def is_supported():
```

Comment:
Is `dgl.cuda.nccl` included in the API doc? We could say the package is in experimental stage.

Reply:
It's not exposed to users directly for now and used in `dgl.nn.NodeEmbedding` and `dgl.optim.SparseAdam` only, where we have a docstring saying it's experimental.